crypto/sm4/asm/vpsm4-armv8.pl

```diff
@@ -1505,12 +1505,12 @@ ()
 .check_dec${std}:
 	// encryption:1 decryption:0
 	cmp $enc,1
-	b.eq .prcess_last_2blks${std}
+	b.eq .process_last_2blks${std}
 	mov @vtmp[0].16B,@tweak[1].16b
 	mov @tweak[1].16B,@tweak[2].16b
 	mov @tweak[2].16B,@vtmp[0].16b
 
-.prcess_last_2blks${std}:
+.process_last_2blks${std}:
```

Comment:
Was there a reason for the label name change here?

Reply:
I think it is a typo, so I fix it by the way.